dtfs-central-api/src/v1/routes/utilisation-reports-routes.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const validation = require('../validation/route-validators/route-validators');
+const handleExpressValidatorResult = require('../validation/route-validators/express-validator-result-handler');
+const {
+  getUtilisationReportsReconciliationSummary,
+} = require('../controllers/utilisation-report-service/get-utilisation-reports-reconciliation-summary.controller');
+
+const utilisationReportsRouter = express.Router();
+
+/**
+ * @openapi
+ * /utilisation-reports/reconciliation-summary/:submissionMonth:
+ *   get:
+ *     summary: |
+ *       Utilisation report reconciliation summary for the specified submission
+ *       month
+ *     tags: [UtilisationReport]
+ *     description: |
+ *       Get a summary of utilisation report reconciliation status for all banks
+ *       in the specified report submission month
+ *     responses:
+ *       200:
+ *         description: OK
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: array
+ *               items:
+ *                 type: object
+ *                 required:
+ *                   - bank
+ *                   - statusCode
+ *                 properties:
+ *                   reportId:
+ *                     type: string
+ *                     description: |
+ *                       The MongoDB '_id' of the associated report (if received)
+ *                       from the 'utilisationReports' collection
+ *                   bank:
+ *                     type: object
+ *                     properties:
+ *                       id:
+ *                         type: string
+ *                       name:
+ *                         type: string
+ *                   statusCode:
```

Comment:
Yeah, fair enough. I was trying to make clear it was our internal code rather than the value that will be displayed, but that's probably unnecessary. Will update 👍